### PR TITLE
Replace nuke with useful information on error page

### PR DESCRIPTION
**Pull request by turbaszek**
This PR replaces nuke asciiart with text about reporting a bug.
As we are no longer using asciiarts this PR removes it.

# Before changes:
Screenshot 2020-10-08 at 11 30 30

# After changes:
Screenshot 2020-10-08 at 11 19 53

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by mik-laj** [comment]
Why is this picture bothering you? In my opinion, this is a nice easter egg.  I am for adding a description, but an additional one, not instead of nuclear.

**Comment by turbaszek** [comment]
> Why do you want to remove nuclear image?

Using nuclear mushroom in professional software is not feasible in my opinion. This is not a lovely little kitten, it's a result of weapon of mass destruction.

**Comment by turbaszek** [comment]
CCing @ryanahamilton as frontend expert

**Comment by ryanahamilton** [comment]
@turbaszek I like these changes. I could certainly brainstorm and suggest some sort of visual flourish to enhance/provide some levity to the page.

(Off camping without a computer for the rest of the week, so won't be able to address until next week)

**Comment by turbaszek** [comment]
> I could certainly brainstorm and suggest some sort of visual flourish to enhance/provide some levity to the page.

That would be awesome and we can do this in follow ups PRs. Probably refreshing error pages would be nice!

> (Off camping without a computer for the rest of the week, so won't be able to address until next week)

Enjoy it! ⛺

File: airflow/www/templates/airflow/traceback.html [hunk 1]
```diff
@@ -23,10 +23,15 @@
   </head>
   <body>
     <div class="container">
-      <h1> Ooops. </h1>
+      <h1> Ooops! </h1>
       <div>
           <pre>
-{{ nukular }}Node: {{ hostname }}
+Something bad had happen.
+Please consider letting us know by creating an <b><a href="https://github.com/apache/airflow/issues/new/choose">bug report using Github</a></b>.
```
Comment (olchas) [inline airflow/www/templates/airflow/traceback.html]:
```suggestion
Please consider letting us know by creating a <b><a href="https://github.com/apache/airflow/issues/new/choose">bug report using Github</a></b>.
```

File: airflow/www/templates/airflow/traceback.html [hunk 2]
```diff
@@ -23,10 +23,15 @@
   </head>
   <body>
     <div class="container">
-      <h1> Ooops. </h1>
+      <h1> Ooops! </h1>
       <div>
           <pre>
-{{ nukular }}Node: {{ hostname }}
+Something bad had happen.
```
Comment (olchas) [inline airflow/www/templates/airflow/traceback.html]:
```suggestion
Something bad has happened.
```

**Review by potiuk — APPROVED** [review]

**Pull request merged**